packages/field-base/src/input-mixin.js

```diff
@@ -164,8 +164,7 @@ export const InputMixin = dedupingMixin(
         // In the case a custom web component is passed as `inputElement`,
         // the actual native input element, on which the event occurred,
         // can be inside shadow trees.
```

Comment:
I think the comment should be moved to the new method as well, as it seems to refer to the use of `composedPath`.